robotbase/src/main/java/frc/robot/commands/algaeRunner/AlgaeRunnerSetSpeed.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.commands.algaeRunner;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Robot;
+
+public class AlgaeRunnerSetSpeed extends Command {
+    private double m_speed;
+
+    public AlgaeRunnerSetSpeed(double speed) {
+        m_speed = speed;
+        addRequirements(Robot.algaeRunner);
+    }
+
+    @Override
+    public void initialize() {
+        Robot.algaeRunner.setAxisSpeed(m_speed);
```

Comment:
We'll want to use the `set` method instead and have a new `AlgaeRunnerSetAxisSpeed` command that will use the setAxisSpeed method